main.go

```diff
@@ -137,6 +141,26 @@ func StartWithOptions(opts *Options) (*Server, error) {
 
 	logger.Debugf("mongod started up and reported a port number after %s", time.Since(startupTime).String())
 
+	// ---------- START OF REPLICA CODE ----------
+	if opts.ShouldUseReplica {
+		mongoCommand := fmt.Sprintf("mongo --port %d --retryWrites --eval \"rs.initiate()\"", opts.Port)
+		//nolint:gosec
+		cmd2 := exec.Command("bash", "-c", mongoCommand)
```

Comment:
You should pass in the arguments individual here (as on line 68) rather than passing a `Sprintf`-formatted string to `exec.Command`. Right now it's OK because you're just templating in a numeric port number, but there's risk here that some future change will introduce a command injection vulnerability.